Makefile

```diff
@@ -37,34 +40,46 @@ help:
 ## Build targets
 
 .PHONY: install
-install:
+install: install-cdo venv
 	@echo "Installing application ..."
-	@-bash -c 'pip install -e .'
+	@-bash -c '${PIP} install -e .'
 	@echo "\nStart service with \`make start'"
 
+.PHONY: install-cdo
+install-cdo:
+	@echo "Installing cdo package ..."
+	@-bash -c "sudo apt-get update"
+	@-bash -c "sudo apt-get install cdo"
+
+.PHONY: install-ci
+install-ci: install-cdo
+	@echo "Installing ci requirements"
+	@-bash -c 'pip install -r requirements.txt'
+	@-bash -c 'pip install -e ".[dev]"'
```

Comment:
`make install-ci` failed the first attempt due to the old `pip` version installed on my workstation. After modifying the `pip` commands to `pip3` solved the problem. I am not sure if the modification is needed for CI actions though.

Reply:
The command works fine in the `actions` environment as it does not have a choice between `pip` and `pip3`.